docs/templates/data_source_parent.rst.jinja

```diff
@@ -38,7 +38,7 @@ the names and descriptions of each of its fields.
    * - Data Dictionary
      - Browse Online
    {%- for resource in source_resources %}
-   * - :ref:`{{ resource.name }}`
+   * - :ref:`{{ resource.sphinx_ref_name }}`
```

Comment:
I ran into a docs build error where Sphinx was unhappy about leading underscores in ref labels.

There were two ways around this:
* don't include the intermediate resources in this list
* figure out how to make legal ref labels with leading underscores

I figured, we went to all this effort to make these tables, we shouldn't hide them - and we already had a somewhat-working way of making legal ref labels (adding the `i` as a separator). So I went for the second option.